packages/react/src/components/Footer/Footer.js

```diff
@@ -29,6 +30,11 @@ const Footer = ({ type }) => {
   const [footerMenuData, setFooterMenuData] = useState([]);
   const [footerLegalData, setFooterLegalData] = useState([]);
 
+  useEffect(() => {
+    // initialize global execution calls
+    globalInit();
```

Comment:
Are we tracking anything in Footer? Or are you planning to add this to every component? Is this how you're seeing executing #391 within the expressive modal as well?

Reply:
Good question. We originally had this added to the DotcomShell, but there would be instances possibly where adopters will pull the `Masthead` and/or `Footer` separately. Ultimately though, this is code that needs to be included if any adopter is using the IBM.com Library. Ideally we would have the adopters just load in the globalInit themselves then trigger it, but that most likely won't be always the case. We can include this as part of the getting started documentation on the website, but again we won't be able to reliably capture everyone doing that on the honor system.

Adding this as an include for the Masthead and Footer would be a good place to capture a large percentage of IBM.com. For those that aren't using the Masthead or the Footer, we would need to ask those adopters to call the globalInit directly.

The other option, to your point, is to include it in every component.

Open to thoughts on this.

@annawen1 @kennylam @andysherman2121 @sannminwin

Reply:
Regardless, I think for now we might be ok with this moving forward in this PR as it will cover all instances of adopters using this. We should possibly add documentation to the README about the globalInit, and/or the cupcake website.

Reply:
I have added this as a huddle topic for next week.